assets/scss/modules/_grid-system.scss

```diff
@@ -79,6 +79,20 @@
     padding: $dp-spaces--lv5;
     color: $dp-color-red;
 
+    a {
+      text-decoration: underline;
+      color: $dp-color-red;
+
+      &:hover {
+        color: $dp-color-darkred;
+      }
+    }
+  }
+
+  .dp-boxshadow--msjuser {
```

Comment:
How about renaming this class like this: `.dp-boxshadow--usermsg` to keep consistency (most classes are named in english)